sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRepository.java

```diff
@@ -243,7 +243,7 @@ public RegistryArtifact getArtifact(String digest) {
      * @throws HttpResponseException thrown if any other unexpected exception is returned by the service.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public PagedIterable<ArtifactManifestProperties> listManifestProperties(ArtifactManifestOrderBy orderBy) {
+    public PagedIterable<ArtifactManifestProperties> listManifestProperties(ArtifactManifestOrder orderBy) {
```

Comment:
Maybe an odd question, why do we have this overload? Normally we've had the base overload that takes the minimum required parameters and the complete overload with all API options (some cases we have additional if the service side API has grown since release).

Reply:
The thought process here was that a lot of people will use this API and may not want to pass in the Context. So yeah more of a popular convenience layer based on service feedback. However, I we can chose to add it later too.

I will add a comment in the APiView and see what architects think.

Reply:
It is also one of the champion scenarios to orderBy a value for example - you may want to delete images from the last 3 days.